fixtures/martin.json

```diff
@@ -17,12 +17,12 @@
                 "power": 170,
                 "DMXconnector": "3-pin (swapped +/-)",
                 "bulb": {
-                    "colorTemperature": 0, // optional
-                    "type": "", // optional. e.g. "LED"
-                    "lumens": 0 // optional
+                    "colorTemperature": 0,
```

Comment:
Please delete those lines completely, as they don't add information when no different from the default values.